mozci/push.py

```diff
@@ -1181,10 +1219,15 @@ def classify_regressions(
         logger.debug(
             f"Got {len(groups_no_confidence)} groups without bugbug confidence"
         )
+        logger.debug(
+            f"Got {len(groups_still_running)} groups still running in the push"
+        )
 
-        # Real failure are groups with likely regressions that were selected with high confidence
-        # AND failing across config
-        real_failures = groups_regressions & groups_relevant_failure & groups_high
+        # Real failures are groups that finished running, with likely regressions that were selected
+        # with high confidence AND failing across config
+        real_failures = (
+            groups_regressions & groups_relevant_failure & groups_high
+        ) - groups_still_running
```

Comment:
We should move these still running groups to the unknown failures list (but take into account that we are not going to do any actions on them as part of #654.